aws/resource_aws_athena_database.go

```diff
@@ -33,6 +33,10 @@ func resourceAwsAthenaDatabase() *schema.Resource {
 				Required: true,
 				ForceNew: true,
 			},
+			"comment": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
```

Comment:
Although, `description` as a parameter might be more self _descriptive_,  the choice of `comment` over that was down to how the database creation [query](https://docs.aws.amazon.com/athena/latest/ug/create-database.html#parameters) is constructed as it was more intuitive for one working with _AWS Athena_.